src/DockAreaTitleBar.cpp

```diff
@@ -195,6 +198,9 @@ void DockAreaTitleBarPrivate::createButtons()
 		TabsMenuButton = new CInvisibleButton();
 	}
 
+	if (CDockManager::configFlags().testFlag(CDockManager::DockAreaHasWidgetActionsMenuButton)){
```

Comment:
In my opinion, using the method "testConfigFlag" is better than direct accessing of "CDockManager::configFlags()" for the sake of _encapsulation_.  (I know that "CDockManager::configFlags().testFlag() is used in many other places and IMHO they all deserve refactoring as well) 
Also, I would respect original coder's preference to have the open curly bracket on the new line.

Reply:
thanks, I did not know that there was a `testConfigFlag` method in this class, will change it.

Forgot to put the curly brace in the new line there, thanks. It would be nice to add a clang formatter hook to this repo to enforce this tiny convention rules.